go/vt/vtgate/executor_test.go

```diff
@@ -1254,6 +1254,51 @@ func waitForColVindexes(t *testing.T, ks, table string, names []string, executor
 	return nil
 }
 
+func waitForVschemaCol(t *testing.T, ks, table string, name string, drop bool, executor *Executor) (*vschemapb.SrvVSchema, *vschemapb.Column) {
```

Comment:
This helper is modeled after some existing helpers, and these similar-looking helps look like they're multiplying. Could be cleaner to replace all the similar-ish wait logic with a single, more generic wait helper (not necessarily in this PR, though). Maybe something like this:

```
vschemaWaiter := makeVschemaWaiter(t, executor.serv, "aa")
_, _ := executor.Execute(... some vschema change ...)
vschema := vschemaWaiter()
// validate that the vschema looks as it should
// run another execute that changes the vschema
vschema = vschemaWaiter()
// validate again
// etc.
```

Where makeVschemaWaiter() might look something like:

```
// makeVschemaWaiter returns a function that gets the current vschema.
func makeVschemaWaiter(t testing.T, s svtopo.Server, cell string) func() *vschemapb.SrvVSchema {
    t.Helper()
	c := make(chan *vschemapb.SrvVSchema, 20)
    sentinel := &vschemapb.SrvVSchema{}
	s.WatchSrvVSchema(context.Background(), cell, func(v *vschemapb.SrvVSchema, err error) {
		c <- v
	})
    return func() *vschemapb.SrvVSchema {
       c <- sentinel
       var lastFound *vschemapb.SrvVSchema
       for {
          select {
            case found := <-c:
                if found == sentinel {
                  return lastFound
                }
                lastFound = found
            case <-time.After(100 * time.Millisecond):
                t.Error("Timeout waiting for vschema update")
                return nil
            }
       }
    }
}
```

Reply:
make sense to me

Reply:
hmm, 2nd thought, as the vschema ddl stuff is to be replaced, i don't want to spend too much renovation on the test case.